parsimonious/expressions.py

```diff
@@ -151,10 +161,32 @@ def _uncached_match(self, text, pos, cache, error):
         if text.startswith(self.literal, pos):
             return Node(self.name, text, pos, pos + len(self.literal))
 
+    def _uncached_match_list(self, token_list, pos, cache, error):
```

Comment:
I'm particularly curious about this method added to Literal, which compares a `.text` attr of `Token` that isn't otherwise mentioned or covered in tests.